sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,160 @@ TypeScript users need to have Node type definitions installed:
+[Azure PowerShell][storage-account-create-ps], or the [Azure CLI][storage-account-create-cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+#### Environment Variables
+
+With a Key Vault and a Storage account, you can fill the rest of the environment variables in the `.env` file used by [dotenv][dotenv]. You can look for more information about other required environment variables in our [sample.env](./sample.env) file.
+
+#### Create KeyVaultAccessControlClient
+
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultAccessControlClient][src-rbac-client]:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultAccessControlClient(vaultUrl, credentials);
+```
+
+#### Create KeyVaultBackupClient
 
-### Configuring your Key Vault
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultBackupClient][src-backup-client]:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultBackupClient(vaultUrl, credentials);
+```
 
 ## Key concepts
 
-## Authenticating with Azure Active Directory
+### KeyVaultRoleDefinition
+
+A Role Definition is a collection of permissions. A role definition defines the operations that can be performed, such as read, write, 
+and delete. It can also define the operations that are excluded from allowed operations.
+
+Role definitions can be listed and specified as part of a Role Assignment.
+
+### KeyVaultRoleAssignment. 
+
+A Role Assignment is the association of a Role Definition to a service principal. They can be created, listed, fetched individually, and deleted.
+
+### KeyVaultAccessControlClient
+
+A [KeyVaultAccessControlClient][src-rbac-client] provides both synchronous and asynchronous operations allowing for management of Role Definitions (instances of `KeyVaultRoleDefinition`) and Role Assignments (instances of `KeyVaultRoleAssignment`).
+
+### KeyVaultBackupClient
+
+A [KeyVaultBackupClient][src-backup-client] provides both synchronous and asynchronous operations for performing full key backups, full key restores, and selective key restores.
+
+### Long Running Operations
```

Comment:
LRO is an abbreviation we use internally and I don't expect end users to be familiar with it. 

```suggestion
### Long running operations
```

Reply:
The rest of the titles are upper-cased like this one 🤔